docs/user-guide/petset.md

```diff
@@ -252,6 +252,19 @@ You can scale a Pet Set by updating the "replicas" field. Note however that the
 1. Create one pet at a time, in order from {0..N-1}, and wait till each one is in [Running and Ready](/docs/user-guide/pod-states) before creating the next
 2. Delete one pet at a time, in reverse order from {N-1..0}, and wait till each one is completely shutdown (past its [terminationGracePeriodSeconds](/docs/user-guide/pods/index#termination-of-pods)) before deleting the next
 
+```shell
+$ kubectl get petset web
+NAME    DESIRED   CURRENT   AGE
+web     2         2         40m
+
+$ kubectl patch petset web -p '{"spec":{"replicas":3}}'
+"web" patched
+
+$ kubectl get petset nginx
+NAME    DESIRED   CURRENT   AGE
+web     3         3         40m
```

Comment:
Can you replace this with a `kubectl get po` and show that there are actually 3 running?